lib/taskana-core/src/test/java/acceptance/task/CreateTaskAccTest.java

```diff
@@ -766,16 +758,18 @@ void testThrowsExceptionIfMandatoryPrimaryObjectReferenceIsNotSetOrIncomplete()
         createObjectReference("COMPANY_A", "SYSTEM_A", "INSTANCE_A", "VNR", null));
     testCreateTask.accept(
         createObjectReference("COMPANY_A", "SYSTEM_A", "INSTANCE_A", null, "1234567"));
-    testCreateTask.accept(createObjectReference("COMPANY_A", "SYSTEM_A", null, "VNR", "1234567"));
-    testCreateTask.accept(createObjectReference("COMPANY_A", null, "INSTANCE_A", "VNR", "1234567"));
+    //    testCreateTask.accept(createObjectReference("COMPANY_A", "SYSTEM_A", null, "VNR",
+    // "1234567"));
+    //    testCreateTask.accept(createObjectReference("COMPANY_A", null, "INSTANCE_A", "VNR",
+    // "1234567"));
```

Comment:
Why commented out?

Reply:
Same as above. but should be deleted not commented.

Because the test expected an exception which is not thrown any more when ObjecReference#System/System_Instance is not set.